go.mod

```diff
@@ -0,0 +1,25 @@
+module github.com/vmware-tanzu/velero-plugin-for-gcp
+
+go 1.13
+
+require (
+	cloud.google.com/go/storage v1.8.0
```

Comment:
Looks like we're bumping to a new version here, which is fine, but since we're doing that let's add a changelog to mention that